<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tree</class>
 <widget class="QDialog" name="Tree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>树可视化</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>336</height>
    </rect>
   </property>
   <widget class="QPushButton" name="BinTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>115</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二叉树</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Hufman">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>115</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>哈夫曼树</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BST">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>115</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二叉搜索树</string>
    </property>
   </widget>
   <widget class="QLabel" name="Problem_Number">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>45</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>题 目 个 数：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Problem">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="Nodes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>95</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>树节点个数：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Node">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>95</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="Problem_Submit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认生成</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BTLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请按照先序输入节点。例：A # B C # # #</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BST_Line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入节点值，并用空格隔开。例：1 2 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_Submit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认生成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AVL_Submit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>平衡构建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Huf_Submit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认生成</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Huf_Line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入权值，并用空格隔开。例：1 2 3 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Heap_Button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>115</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>构造堆</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Heap_Line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入数值，并用空格隔开。例：4 5 6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Min_heap">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>构造小根堆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Max_heap">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>构造大根堆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BST_Submit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>按序构建</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Problem_Type">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>树的遍历</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>树的构造</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>哈弗曼树</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Problem_Types">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>135</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>题 目 类 型 ：</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
